### ISSUE TITLE
PSNR is slightly lower in my experiments

### ISSUE THREAD

**Issue by JingyunLiang** [Zheng222/IMDN#16]
I tested IMDN_X4 using the provided checkpoint and Set5 dataset. The PSNR is 32.19. I also trained the IMDN_X4 from scratch strictly following your code. The best PSNR is 32.14. These two results are slightly lower than the reported 32.21 in Table 5. 

What's the possible reason for the two differences? Do I need to train a model based on your pre-trained checkpoint? (i.e., `--pretrained checkpoints/IMDN_x4.pth`) If so, how was this checkpoint trained?

**Comment by Lininggggggg** [comment]
i have the same issus about this , could you tell me? dear author

**Comment by Lininggggggg** [comment]
what more, why the test time is different so large with different environment ?
how should i to choose the best environment for my workplace?
https://github.com/Zheng222/IMDN/issues/4#issue-506186607
and i found this link but i still have trouble about this?
could you tell me? Thanks!!!

**Comment by Zheng222** [comment]
@JingyunLiang Please use MATLAB to test PSNR values. If you want to further improve the PSNR, please train more steps when the learning rate is 2e-4.

**Comment by Zheng222** [comment]
@Lininggggggg This problem does not occur on my machine. You can test time using [KAIR](https://github.com/cszn/KAIR/blob/23b0d0f717980e48fad02513ba14045d57264fe1/main_challenge_sr.py) and then compare it with our testing time.

**Comment by Lininggggggg** [comment]
OK,thank you very much!!!

**Comment by JingyunLiang** [comment]
Thank u.
